sampleDriver/EvsEnumerator.cpp

```diff
@@ -282,33 +294,52 @@ Return<sp<IEvsCamera_1_0>> EvsEnumerator::openCamera(const hidl_string& cameraId
     }
 
     // Is this a recognized camera id?
-    CameraRecord *pRecord = findCameraById(cameraId);
+    CameraRecord* pRecord = findCameraById(cameraId);
```

Comment:
this change not needed

Reply:
google doesn't have coding style, pointers and references are without rules

Reply:
https://google.github.io/styleguide/cppguide.html  In any means - we stick to the original style of the file and don't change it. And not adding non functional style changes.